library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -234,28 +233,26 @@ class ChuckerInterceptor internal constructor(
         private val transaction: HttpTransaction
     ) : TeeSource.Callback {
 
-        override fun onClosed(file: File, totalBytesRead: Long) {
-            val buffer = try {
-                readResponseBuffer(file, response.isGzipped)
-            } catch (e: IOException) {
-                null
-            }
+        override fun onClosed(file: File?, totalBytesRead: Long) {
+            val buffer = file?.let { readResponseBuffer(it, response.isGzipped) }
```

Comment:
I would suggest to avoid `?.let { }` here and use just `if (file != null) {...` to not introduce additional useless variable multiple times (as this fun will be called for each transaction).
Suggestion is based on [this article](https://medium.com/mobile-app-development-publication/kotlin-dont-just-use-let-7e91f544e27f) and recommendation 1.1 from the mentioned article.